recipe/meta.yaml

```diff
@@ -1,53 +1,52 @@
-{% set version = "2.0.2" %}
+{% set version = "0.8.4" %}
 
 package:
   name: mistune
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/m/mistune/mistune-{{ version }}.tar.gz
-  sha256: 6fc88c3cb49dba8b16687b41725e661cf85784c12e8974a29b9d336dd596c3a1
+  sha256: 59a3429db53c50b5c6bcc8a07f8848cb00d7dc8bdb431a4ab41920d201d4756e
 
 build:
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 1000
```

Comment:
Maybe it is worth explaining, where `main-0.8.4` comes from and why we seem to be downgrading here and why we're bumping the build number to 1000. Since we're bumping the build no. the trigger below is probably unnecessary.

Reply:
The latest version before 2.0.0 was v0.8.4 https://github.com/AnacondaRecipes/mistune-feedstock/blob/a9bcbc25763bc4b5b8e15c81bc084250aed59a9b/recipe/meta.yaml and the build number was 1000. As we need only python 3.10 support for ppc64le I rebuild it in the main-0.8.4 branch.

nbconvert greater than v6.1 requires `mistune <2` https://github.com/AnacondaRecipes/nbconvert-feedstock/blob/aa99e7e9c88e8ca00a4ac043083a6b330f026042/recipe/meta.yaml#L27 but we are missingthe  mistune 0.8.4 artifact for python 3.10 on ppc64le.

Reply:
But then it seems appropriate to me to create the main-0.8.4 branch from that commit instead of trimming what looks like master to look like the previous version.

Reply:
You are right. It was my mistake